### PR TITLE
Support ethsign signatures

### PR DESCRIPTION
**Pull request by vkgnosis**
### Test Plan
unit tests are testing both signatures

### CODE REVIEW

File: database/sql/V008__contracts_upgrade.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TYPE SigningScheme ADD VALUE 'ethsign';
```
Comment (fleupold) [inline database/sql/V008__contracts_upgrade.sql:1]: nit: the migration name should probably be "add ethsign signature or the like" (unless it goes in the same file as the contract upgrade)

File: model/src/lib.rs [hunk 2]
```diff
@@ -37,6 +38,24 @@ fn hashed_eip712_message(domain_separator: &DomainSeparator, struct_hash: &[u8;
...
 }
 
+fn hashed_ethsign_message(domain_separator: &DomainSeparator, struct_hash: &[u8; 32]) -> [u8; 32] {
+    let mut message = [0u8; 60];
+    message[..28].copy_from_slice(b"\x19Ethereum Signed Message:\n32");
+    message[28..].copy_from_slice(&hashed_eip712_message(domain_separator, struct_hash));
+    signing::keccak256(&message)
+}
+
+fn signing_message(
```
Comment (fleupold) [inline model/src/lib.rs:48]: naming nit: to be more inline with the ethsign/eip712 methods maybe `hashed_signing_message()` (I kind of like the `digest` word e.g. `ethsign_message_digest` in the previous version).

**Review by fleupold — APPROVED** [review]
Maybe we can change the e2e test to use one order of each signature kind to make sure it works with the smart contracts?

**Review by bh2smith — APPROVED** [review]

**Comment by vkgnosis** [comment]
I am merging this so that there is only a single, up to date contracts upgrade PR. Both PRs have been reviewed individually already.

**Pull request merged**